samples/server/petstore/jaxrs-resteasy/eap-java8/pom.xml

```diff
@@ -34,23 +46,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>1.6.0</version>
-                <executions>
-                    <execution>
-                        <id>Play Test</id>
-                        <phase>integration-test</phase>
-                        <goals>
-                            <goal>exec</goal>
-                        </goals>
-                        <configuration>
-                            <executable>./mvn_test_jdk8_only.sh</executable>
```

Comment:
I think we need to use this (./mvn_test_jdk8_only.sh) to only test in Java8 for this particular project as the test environment (CircleCI) uses both OpenJDK 7 & 8 to run the tests.